srcpkgs/chronograf/template

```diff
@@ -21,15 +21,17 @@ make_dirs="${_chronograf_homedir} 0755 _chronograf _chronograf"
 case "$XBPS_TARGET_MACHINE" in
 	i686*) broken="yarn not available" ;;
 	ppc*) broken="ftbfs in some js module" ;;
+	aarch64-musl) broken="aarch64-linux-musl-g++: error: unrecognized command-line option '-m64'" ;;
+	armv6l-musl) broken="arm-linux-musleabihf-g++: error: unrecognized command-line option '-m64'" ;;
```

Comment:
it's not really a good idea to suddenly remove a package from some repositories, this would require manual intervention into the repositories and people with chronograf on these architectures wouldn't be able to update. Can we just patch it out?

Reply:
is setting nocross for musl archs an option?

Reply:
no, it has the same issue